tensorflow/python/ops/math_ops.py

```diff
@@ -4857,6 +4857,68 @@ def sparse_segment_sum(
         name=name,
     )
 
+@tf_export("sparse.sampled_addmm")
+def sampled_addmm(indices,
+    values,
+    dense_shape,
+    mat1,
+    mat2,
+    beta=1.0,
+    alpha=1.0
+):
+  sparse_rank = len(dense_shape)
+  sparse_num_batches = dense_shape[0] if sparse_rank == 3 else 1
```

Comment:
You _could_ have this work for multiple batch dimensions, using the last two as the matrix dimensions.

Reply:
Still need to do some more testing with multiple batch dimensions, but I removed this logic checking for 1 or no batch dims